src/components/Voting.js

```diff
@@ -0,0 +1,33 @@
+import { useEffect, useState } from "react";
+import { PatchArticleVotes } from "./api-logic";
+
+const Voting = ({ SingleArticle, article_id }) => {
+  const [vote, setVote] = useState(0);
+  const [appendedVote, setAppendedVote] = useState(SingleArticle.votes);
+  useEffect(() => {
+    PatchArticleVotes(article_id, vote)
+      .then((response) => response.json())
+      .then((response) => setAppendedVote(response.votes));
+  }, [article_id, vote]);
+
+  const incrementVote = (e) => {
+    setVote(1);
+  };
+  const decrementVote = (e) => {
+    setVote(-1);
+  };
+
+  return (
+    <section id="vote-section">
+      <h4>Votes: {appendedVote}</h4>
+      <button className="vote-button" onClick={incrementVote}>
```

Comment:
As a bonus, I'd look at including some conditional styling here to make the buttons disabled when pressed. That should prevent people flicking between upvote and downvote, causing a lot of patch requests